test/unit/runner.spec.js

```diff
@@ -385,6 +385,40 @@ describe('Runner', function() {
     });
   });
 
+  describe('.run(fn)', function() {
+    it('should emit "retryable fail" when a retryable test fails', function(done) {
+      var retries = 2;
+      var retryableFails = 0;
+      var ERR = new Error('bear error');
+
+      var TEST = new Test('im a test about bears', function() {
+        if (retryableFails < retries) {
+          throw ERR;
+        }
+      });
+
+      TEST.retries(retries);
+      suite.retries(retries);
```

Comment:
`suite.retries(retries);` is enought for tests. 
IMO `TEST.retries(retries);` not unneccesary here.

Reply:
Thanks for the feedback. I agree and have updated the PR.